migrations/back_populate_uuids.py

```diff
@@ -0,0 +1,176 @@
+
+    logger.info(f"Starting update process with Start datetime {start_datetime} and End datetime {end_datetime}")
+
+    update_mongo(config=config, updated_at=updated_at_datetime)
+
+    update_dart(config=config, start_datetime=start_datetime, end_datetime=end_datetime)
+
+
+def validate_args(
+    config: Config, s_start_datetime: str, s_end_datetime: str, updated_at: str
+) -> Tuple[datetime, datetime, datetime]:
+    base_msg = "Aborting run: "
+    if not config:
+        msg = f"{base_msg} Config required"
+        logger.error(msg)
+        raise Exception(msg)
+
+    if not valid_datetime_string(s_start_datetime):
+        msg = f"{base_msg} Expected format of Start datetime is YYMMDD_HHmm"
+        logger.error(msg)
+        raise Exception(msg)
+
+    if not valid_datetime_string(s_end_datetime):
+        msg = f"{base_msg} Expected format of End datetime is YYMMDD_HHmm"
+        logger.error(msg)
+        raise Exception(msg)
+
+    if not valid_datetime_string(updated_at):
+        msg = f"{base_msg} Expected format of updated_at datetime is YYMMDD_HHmm"
+        logger.error(msg)
+        raise Exception(msg)
+
+    start_datetime = datetime.strptime(s_start_datetime, MONGO_DATETIME_FORMAT)
+    end_datetime = datetime.strptime(s_end_datetime, MONGO_DATETIME_FORMAT)
+    updated_at_datetime = datetime.strptime(updated_at, MONGO_DATETIME_FORMAT)
+
+    if start_datetime > end_datetime:
+        msg = f"{base_msg} End datetime must be greater than Start datetime"
+        logger.error(msg)
+        raise Exception(msg)
+
+    return start_datetime, end_datetime, updated_at_datetime
+
+
+def update_mongo(config: Config, updated_at: datetime) -> None:
+    with create_mongo_client(config) as client:
+        mongo_db = get_mongo_db(config, client)
+
+        samples_collection = get_mongo_collection(mongo_db, COLLECTION_SAMPLES)
+
+        counter = 0
+        for mysql_sample in mysql_sample_generator(
+            config=config,
+            query=f"SELECT * FROM lighthouse_sample WHERE updated_at > '{updated_at.strftime('%Y-%m-%d %H:%M')}'",
+        ):
+            mlwh_sample_uuid = mysql_sample.get(MLWH_LH_SAMPLE_UUID)
+
+            if mlwh_sample_uuid is None:
+                continue
+
+            mongo_sample = samples_collection.find_one_and_update(
+                filter={
+                    FIELD_MONGODB_ID: ObjectId(mysql_sample.get(MLWH_MONGODB_ID)),
+                    FIELD_LH_SAMPLE_UUID: {
+                        "$ne": mlwh_sample_uuid,
+                    },
+                },
+                update={
+                    "$set": {
+                        FIELD_LH_SAMPLE_UUID: mlwh_sample_uuid,
+                        UUID_UPDATED: True,
+                        FIELD_UPDATED_AT: datetime.utcnow(),
+                    }
+                },
+            )
+
+            if mongo_sample is not None:
+                counter += 1
+
+            if (counter % 5000) == 0:
```

Comment:
Whats this for? If the counter is divisible by 5000?

Reply:
It's just an output to track how progress is going since it might be a large dataset.